/* ============================

    该文件, 用于不同平台下, 使用google protocol buffer
    编译工作, 编译能在不同平台下使用的头文件和源文件

 ============================ */


package iegad.msg;


message basic_msg {
// the message of tcp transfer;
    required int32  msg_type = 1;	// sub_service id
    required int32  msg_flag = 2;	// forward extended
    required bytes  msg_bdstr = 3;	// sub_service builder
};